ci-build.sh

```diff
@@ -7,7 +7,7 @@ BUILD_NUMBER="${BUILD_NUMBER:-${DRONE_BUILD_NUMBER}}"
 PORT=$((${BUILD_NUMBER} + 1025))
 BUILD_NUMBER="${BUILD_NUMBER:-local}"
 START_INSTANCE="docker run "
-DOCKER_HOST_NAME=172.17.0.1
+: "${DOCKER_HOST_NAME:=172.17.0.1}"
```

Comment:
Should we change the syntax to match the other similar declarations in this code block?
```suggestion
DOCKER_HOST_NAME="{DOCKER_HOST_NAME:-127.17.0.1}"
```

Reply:
I prefer the more concise form simply because it removes one more opportunity for a typo ruining everything and doesn't repeat the same name twice, but it's not a particularly strong preference.